src/tapis-ui/components/jobs/JobLauncher/FileInputs.tsx

```diff
@@ -79,13 +78,9 @@ const FileInputField: FieldArrayComponent<FileInput> = ({
   );
 };
 
-type FileInputsProps = {
-  appInputs: Array<FileInput>;
-};
-
-const FileInputs: React.FC<FileInputsProps> = ({ appInputs }) => {
-  const name: FieldArrayPath<ReqSubmitJob> = 'fileInputs';
+type FileInput = TFieldArray<Required<ReqSubmitJob>, 'fileInputs'>;
 
+const FileInputs: React.FC<{ appInputs: FileInput[] }> = ({ appInputs }) => {
```

Comment:
FileInput is actually a different type. There's:

```
import { FileInput } from '@tapis/tapis-typescript-apps';
```

and 

```
import { InputFileSpec } from `@tapis/tapis-typescript-jobs`
```

Where appInputs is used at the JobLauncher level to map an app def into a default job submit value. When the Jobs service changes, these object types will become very different.